### PR TITLE
Feature/new logs audit [OSF-5945]

### PR DESCRIPTION
**Pull request by pattisdr**
Issue https://openscience.atlassian.net/browse/OSF-5945

Moved from https://github.com/CenterForOpenScience/osf.io/pull/5394

# Purpose

Audits new logs on My Projects page.  For the most part, tries to match current logs.  As a general rule, files will provide links whether or not the resource has been deleted. (If it has been deleted, the link will return a Gone or Not Found).  Logs referencing nodes, however, will not provide links if the resource has been deleted.   

# Changes

Numerous small fixes to logs that appear in My Projects page - no change to project organizer logs.  Log phrasing changed, links added, etc.  For full list of changes see google doc link in Issue https://openscience.atlassian.net/browse/OSF-5945

### CODE REVIEW

**Comment by pattisdr** [comment]
@hmoco, do you think I can move this to ready to merge?

**Comment by hmoco** [comment]
@pattisdr yep!

**Comment by pattisdr** [comment]
Great. Thanks again for reviewing this for me.

File: website/static/js/logTextParser.js [hunk 1]
```diff
@@ -243,25 +325,64 @@ var LogPieces = {
     // The new title of node involved
     title_new: {
         view: function (ctrl, logObject) {
-            return returnTextParams('title_new', 'a title', logObject);
+            var url = null;
+            var nodeObject = logObject.embeds.original_node;
+
+            if (paramIsReturned(nodeObject, logObject && nodeObject.data)){
+                 if (nodeObject.data.links && nodeObject.data.attributes) {
```
Comment (sloria) [inline website/static/js/logTextParser.js]: Why is it necessary to check `.data.attributes` here? If you can remove that check, you could use `lodash.get` to get `.data.links.html`.

File: website/static/js/logTextParser.js [hunk 2]
```diff
@@ -211,26 +246,73 @@ var LogPieces = {
...
     template: {
         view: function (ctrl, logObject) {
             var template_node = logObject.embeds.template_node;
+
             if(paramIsReturned(template_node, logObject)){
                 return m('a', {href: template_node.data.links.html}, template_node.data.attributes.title);
             }
+
+            var templateFromParams = logObject.attributes.params.template_node;
+                if (paramIsReturned(templateFromParams, logObject && templateFromParams.title)){
```
Comment (sloria) [inline website/static/js/logTextParser.js]: Minor; this block is overindented.

File: website/static/js/project-organizer.js [hunk 3]
```diff
@@ -53,9 +53,9 @@ function _poTitleColumn(item) {
  * @private
  */
 function _poContributors(item) {
-    var contributorList = item.data.embeds.contributors.data;
-    if(!contributorList){
-        return '';
+    var contributorList = [];
+    if (item.data && item.data.embeds && item.data.embeds.contributors) {
+        contributorList = item.data.embeds.contributors.data;
```
Comment (sloria) [inline website/static/js/project-organizer.js]: You can use `lodash.get` here.

File: website/static/js/myProjects.js [hunk 4]
```diff
@@ -287,7 +287,10 @@ function _formatDataforPO(item) {
     //
     //Sets for filtering
     item.tagSet = new Set(item.attributes.tags || []);
-    var contributors = item.embeds.contributors.data || [];
+    var contributors = [];
+    if (item.embeds && item.embeds.contributors) {
+        contributors = item.embeds.contributors.data;
```
Comment (sloria) [inline website/static/js/myProjects.js]: Same here; use `lodash.get`.

**Comment by sloria** [comment]
Pass finished; just a few minor changes, and this is good to merge.

**Comment by pattisdr** [comment]
@sloria, requested changes made.

**Pull request merged**